bridges/snowbridge/pallets/ethereum-client/src/tests.rs

```diff
@@ -129,6 +128,23 @@ pub fn compute_domain_bls() {
 	});
 }
 
+#[test]
+pub fn may_refund_call_fee() {
+	let finalized_update = Box::new(load_next_finalized_header_update_fixture());
+	let sync_committee_update = Box::new(load_sync_committee_update_fixture());
+	new_tester().execute_with(|| {
+		// Not free, smaller than the allowed free header interval
+		assert_eq!(
+			EthereumBeaconClient::check_refundable(&finalized_update.clone(), 8190),
```

Comment:
It might be clearer if you use `finalized_update.finalized_header.slot + some_offset`

Reply:
Agreed, updated here: https://github.com/paritytech/polkadot-sdk/pull/5201/commits/ab8739cff56a96ad838020a451d9809b7f617d15#diff-ea6025b3cac6719f750d1f0065ba30d70a16d386a5ce374e4a6b79299f48a3eeR136